alfalfa_worker/dispatcher.py

```diff
@@ -135,3 +161,38 @@ def run(self):
             except BaseException as e:
                 tb = traceback.format_exc()
                 self.logger.info("Exception caught in dispatcher.run: {} with {}".format(e, tb))
+
+    def start_job(self, job_name, parameters):
+        """Start job in thread by Python class path"""
+        klazz = self.find_class(job_name)
```

Comment:
Yes, that is the goal I think. Right now I didn't want to modify the web server to send the corresponding messages with job names.

Reply:
is the plan to 
1. merge in updates to the worker/jobs w/out changes to web, and then
2. coordinate web updates and worker cleanup?

This seems reasonable to me provided there isn't too much overhead to avoid changes in the worker.  

@kbenne at what point would you be able to start integrating w/BOPtest?  1 or 2 above?